release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2550,6 +2602,78 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2057209[*BZ#2057209*])
+
+* In the default ZTP Argo CD configuration, cluster names cannot begin with `ztp`. Using names starting with `ztp` for clusters deployed with Zero Touch Provisioning (ZTP) results in provisioning not completing. As a workaround, ensure that either cluster names do not start with `ztp`, or adjust the Argo CD policy application namespace to a pattern that excludes the names of your clusters but still matches your policy namespace. For example, if your cluster names start with `ztp`, change the pattern in the Argo CD policy app configuration to something different, like `ztp-`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2049154[*BZ#2049154*])
+
+* During a spoke cluster upgrade, one or more reconcile errors is recorded in the container log. The number of errors corresponds to the number of child policies. The error causes no noticeable impact to the cluster. The following is an example of the reconcile error:
++
+[source,terminal]
+----
+2022-01-21T00:14:44.697Z        INFO    controllers.ClusterGroupUpgrade Upgrade is completed
+2022-01-21T00:14:44.892Z        ERROR   controller-runtime.manager.controller.clustergroupupgrade       Reconciler error        {"reconciler group": "ran.openshift.io", "reconciler kind": "ClusterGroupUpgrade", "name": "timeout", "namespace": "default", "error": "Operation cannot be fulfilled on clustergroupupgrades.ran.openshift.io \"timeout\": the object has been modified; please apply your changes to the latest version and try again"}
+sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
+        /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.2/pkg/internal/controller/controller.go:253
+sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
+        /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.2/pkg/internal/controller/controller.go:214
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2043301[*BZ#2043301*])
+
+* In {product-title} 4.10, `linxptp` daemon usage has more than doubled since earlier measurements in version 4.9. For an ordinary clock PTP configuration using the Intel e810 NIC, CPU usage approaches 45 mc. No faults are recorded in the logs. These results are found with the following NIC configuration:
++
+[source,terminal]
+----
+NIC
+ethtool -i ens1f0
+driver: ice
+version: 4.18.0-305.34.2.rt7.107.el8_4.x
+firmware-version: 3.00 0x80008278 1.2992.0
+expansion-rom-version:
+bus-info: 0000:8a:00.0
+supports-statistics: yes
+supports-test: yes
+supports-eeprom-access: yes
+supports-register-dump: yes
+supports-priv-flags: yes
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2050994[*BZ#2050994*])
```

Comment:
There is nothing a user can do in this case and the net increase of CPU usage is small. No release note is needed for this (BZ 2050994)

Reply:
follow up from SME directed that this BZ should be removed